inventory-generation/tower_jobs_schedules/templates/do500-tower-management-host.yaml.j2

```diff
@@ -89,15 +104,24 @@ ansible_tower:
         title: {% raw %}"{{ shutoff_reminder.title }}"{% endraw %}
         list_of_users: {% raw %}"{{ internal.list_of_users | default([]) }}"{% endraw %}
         list_of_mail_cc: {% raw %}"{{ list_of_mail_cc | default([]) }}"{% endraw %}
-    - name: "{{ customer_engagement }}-shutoff"
-      description: "Shutoff e-mail for the {{ customer_engagement }} engagement"
+    - name: "{{ customer_engagement }}-environment-termination"
+      description: "Environment termination for the {{ customer_engagement }} engagement"
       {% raw -%}
       rrule: "{{ archive_date | parse_datetime | replace_datetime(hour=5) | to_rrule(timezone=timezone) }}"
       {%- endraw %}
-      unified_job_template: "{{ customer_engagement }}-email-notify-list-of-users"
+      unified_job_template: "{{ customer_engagement }}-termination-workflow"
       enabled: {{ enable_notifications | default(false) }}
       extra_data:
         body: {% raw %}"{{ shutoff.body }}"{% endraw %}
         title: {% raw %}"{{ shutoff.title }}"{% endraw %}
-        list_of_users: {% raw %}"{{ internal.list_of_users | default([]) }}"{% endraw %}
+        list_of_users: {% raw %}"{{ full.list_of_users | default([]) }}"{% endraw %}
         list_of_mail_cc: {% raw %}"{{ list_of_mail_cc | default([]) }}"{% endraw %}
+        openshift_remove_objects:
+          - name: "gitlab-app-{{ project_id }}"
+            kind: "Application"
+            namespace: "lodestar-argo-cd"
+            api_version: argoproj.io/v1alpha1
+          - name: "do500.ocp4.dev-{{ project_id }}"
```

Comment:
`dev` needs to be set to `engagement_region | default('dev')`

Reply:
Fixed